prac_10/wiki.py

```diff
@@ -0,0 +1,28 @@
+import wikipedia
+
+
+def get_page_summary(title):
+    """
+    Get and print the title, summary, and URL of a Wikipedia page.
+
+    Parameters:
+    - title: The title or search phrase for the Wikipedia page.
+    """
+    try:
+        page = wikipedia.page(title)  # Autosuggest=False was removed due to pycharm's suggestion
+        print("Title:", page.title)
+        print("Summary:", page.summary)
+        print("URL:", page.url)
+    except wikipedia.DisambiguationError as e:
+        print("Disambiguation Error: ", e.options)
```

Comment:
could you give this a more meaningful error message e.g. "title/phrase" may refer to: